apps/web-giddh/src/app/inventory/components/aside-pane/aside-pane.components.html

```diff
@@ -8,16 +8,16 @@ <h3>Create</h3>
     <button id="back" (click)="backButtonPressed()" *ngIf="hideFirstScreen">Back</button>
 
     <div class="aside-body flexy-child vertical-align-middle aside-panel-ledger" *ngIf="!hideFirstScreen" vr-item>
-        <div class="pd-b3 text-center">
+        <div class="pb-3 text-center">
```

Comment:
Class name getting replaced. Please check

Reply:
Confirmed from @aditi-tiwari17 it is required.